set.mm

```diff
@@ -524204,6 +524220,16 @@ References are made to the second edition (1927, reprinted 1963) of
 
 $)
 
+  ${
+    bj-sbievv.nfx $e |- F/ x ps $.
+    bj-sbievv.nfy $e |- F/ y ph $.
+    bj-sbievv.is $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Version of ~ sbie with a second non-freeness hypothesis and shorter
+       proof.  (Contributed by BJ, 18-Jul-2023.) $)
+    bj-sbievv $p |- ( [ y / x ] ph <-> ps ) $=
+      ( wsb weq wi wal sb6f equsal bitri ) ACDHCDIAJCKBACDFLABCDEGMN $.
+  $}
+
```

Comment:
would like to have this theorem also in main, because it fits to the "sbie*" series

Reply:
Use sbie instead.  It does not rely on F/ y ph, and has the same axiom footprint.  In my eyes we should not move it to main.